### PR TITLE
Triage doc updates

### PR DESCRIPTION
**Pull request by IgorMinar**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: TRIAGE_AND_LABELS.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ There are few components which are cross-cutting. They don't have
 a clear location in the source tree. We will treat them as a component
 even thought no specific source tree is associated with them.
 
-* `comp: documentation`: `@naomiblack`
+* `comp: docs`: `@naomiblack`
```
Comment (vicb) [inline TRIAGE_AND_LABELS.md]: `compX` IIRC, other should also be updated

File: TRIAGE_AND_LABELS.md [hunk 2]
```diff
@@ -53,11 +53,11 @@ What kind of problem is this?
...
 ## Caretaker Triage Process
 
-It is the caretaker's responsibility to assign `comp:  *` and `type: *`
-to each new issue as they come in. The reason why we limit the 
-responsibility of the caretaker to these two labels is that it is 
-unlikely that without domain knowledge the caretaker could add any 
-additional labels of value.
+It is the caretaker's responsibility to assign `comp:  *` to each new
+issue as they come in. The reason why we limit the responsibility of the
+caretaker to these one label is that it is likely that without domain
```
Comment (vicb) [inline TRIAGE_AND_LABELS.md]: these one label -> this category ? (or label*s* probably?)

File: TRIAGE_AND_LABELS.md [hunk 3]
```diff
@@ -68,11 +68,39 @@ process for their component.
 It will be up to the component owner to determine the order in which the
 issues within the component will be resolved.
 
+Several owners have adopted the issue categorization based on
```
Comment (vicb) [inline TRIAGE_AND_LABELS.md]: I would prefer that we don't commit on that before we discuss... but I'm ok

File: TRIAGE_AND_LABELS.md [hunk 4]
```diff
@@ -68,11 +68,39 @@ process for their component.
...
+[user pain](http://www.lostgarden.com/2008/05/improving-bug-triage-with-user-pain.html)
+used by Angular 1. In this system each issue is assigned frequency and
```
Comment (vicb) [inline TRIAGE_AND_LABELS.md]: **every** issue ?

Comment (IgorMinar) [inline TRIAGE_AND_LABELS.md]: done

File: TRIAGE_AND_LABELS.md (hunk 3 above)
Comment (IgorMinar) [inline TRIAGE_AND_LABELS.md]: This is true. Me, Tobias and you have adopted this system.

File: TRIAGE_AND_LABELS.md (hunk 1 above)
Comment (IgorMinar) [inline TRIAGE_AND_LABELS.md]: it should be up to date

**Pull request merged**